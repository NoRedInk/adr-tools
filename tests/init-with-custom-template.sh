echo custom template > init.md
ADR_INIT_TEMPLATE=init.md adr init
ls doc/adr
cat doc/adr/0001-record-architecture-decisions.md
