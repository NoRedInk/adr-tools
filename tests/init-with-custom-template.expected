echo custom template > init.md
ADR_INIT_TEMPLATE=init.md adr init
doc/adr/0001-record-architecture-decisions.md
ls doc/adr
0001-record-architecture-decisions.md
cat doc/adr/0001-record-architecture-decisions.md
custom template
